wordpress-mobile/WordPress-iOS
# JP to WP Migration - Complete notifications permissions and success screen UI

Fixes #19570

This PR adds missing UI elements to the notifications permission and success screen. This PR also:

- changes the font color of the secondary description to Gray50 (light) / Gray10(dark) in `MigrationHeaderView`
- replaces the images in `MigrationHeaderView` for the Notifications and Success screen, setting now the right ones.

notifications | success
-- | --
<img height=450 src=https://user-images.githubusercontent.com/34376330/200878812-23f386ac-586f-4ccb-a5d9-03bb1488b831.png> |  <img height=450 src=https://user-images.githubusercontent.com/34376330/201432623-c66ccc4f-380c-4958-a4fd-a9edab15605d.png>


To test:

**pre-requisite** 
make sure that the notifications permissions for the Jetpack app on your device or simulator have not already been set, otherwise:
- delete the app
- re-install from this branch
- log in
- kill the app
- start the testing steps below.

**testing steps**
- checkout this branch
- in `JetpackWindowManager`, set `shouldShowMigrationUI` to `true`
- build/run the `Jetpack` target. The migration flow should start
- Verify that the secondary description in the welcome screen ("we found your site/sites etc...") is gray
- Tap Continue
- Verify that the following screen (notifications permissions) matches the UI (example screenshot above, refer to the related design P2 post)
- tap any button (note: if you tap continue, the notifications permission alert will be presented: in this case, just select a choice)
- make sure the success (all done) screen appears
- make sure the image at the top matches the designs
- 
NOTE: this PR does not finalize the success screen. It will happen in a separate PR.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
